src/transactions/feed/JumpstartFeedItem.tsx

```diff
@@ -0,0 +1,127 @@
+import BigNumber from 'bignumber.js'
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { StyleSheet, Text, View } from 'react-native'
+import { HomeEvents } from 'src/analytics/Events'
+import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
+import { hideHomeBalancesSelector } from 'src/app/selectors'
+import TokenDisplay from 'src/components/TokenDisplay'
+import Touchable from 'src/components/Touchable'
+import { navigate } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import { useSelector } from 'src/redux/hooks'
+import { getFeatureGate } from 'src/statsig'
+import { StatsigFeatureGates } from 'src/statsig/types'
+import colors from 'src/styles/colors'
+import { typeScale } from 'src/styles/fonts'
+import { Spacing } from 'src/styles/styles'
+import variables from 'src/styles/variables'
+import { useTokenInfo } from 'src/tokens/hooks'
+import TransactionFeedItemImage from 'src/transactions/feed/TransactionFeedItemImage'
+import { TokenTransactionTypeV2, TokenTransfer } from 'src/transactions/types'
+
+interface Props {
+  transfer: TokenTransfer
+}
+
+function JumpstartFeedItem({ transfer }: Props) {
+  const { t } = useTranslation()
+  const { amount, type } = transfer
+
+  const openTransferDetails = () => {
+    navigate(Screens.TransactionDetailsScreen, { transaction: transfer })
+    ValoraAnalytics.track(HomeEvents.transaction_feed_item_select)
+  }
+
+  const tokenInfo = useTokenInfo(amount.tokenId)
+  const showTokenAmount = !amount.localAmount && !tokenInfo?.priceUsd
+  const colorStyle = new BigNumber(amount.value).isPositive() ? { color: colors.primary } : {}
+
+  const hideHomeBalanceState = useSelector(hideHomeBalancesSelector)
+  const hideBalance =
+    getFeatureGate(StatsigFeatureGates.SHOW_HIDE_HOME_BALANCES_TOGGLE) && hideHomeBalanceState
+
+  const subtitle =
+    type === TokenTransactionTypeV2.Received
+      ? t('feedItemJumpstartReceivedSubtitle')
+      : t('feedItemJumpstartSentSubtitle')
+
+  return (
```

Comment:
reading this component, it is so similar to the existing TransferFeedItem...did you consider sharing code between the components? in particular, i think the whole render block could be shared? it seems like the only areas of customisation are the image, title, and subtitle?

Reply:
Yes! I considered it but the code was a bit ugly because how the `TransactionFeedItemImage` was used.
And I went with the quick solution, but now I changed `TransactionFeedItemImage` implementation, so I think I can merge this feed item to the transfer one.
Thanks for the call out!